chart/templates/back/deployment.yaml

```diff
@@ -0,0 +1,171 @@
+        {{- range $key, $value := . }}
+        {{ $key }}: {{ $value | quote }}
+        {{- end }}
+        {{- end }}
+      labels:
+        {{- include "kyoo.labels" (dict "context" . "component" .Values.back.name "name" .Values.back.name) | nindent 8 }}
+        {{- with (mergeOverwrite (deepCopy .Values.global.podLabels) .Values.back.podLabels) }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.back.imagePullSecrets | default .Values.global.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.global.securityContext }}
+      securityContext:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "kyoo.back.serviceAccountName" . }}
+      initContainers:
+        - name: migrations
+          image: {{ .Values.back.kyoo_migrations.image.repository | default (printf "%s/kyoo_migrations" .Values.global.image.repositoryBase) }}:{{ default (include "kyoo.defaultTag" .) .Values.back.kyoo_migrations.image.tag }}
+          imagePullPolicy: {{ default .Values.global.image.imagePullPolicy }}
+          args:
+            {{- with .Values.back.kyoo_migrations.extraArgs }}
+              {{- toYaml . | nindent 12 }}
+            {{- end }}
+          env:
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  key: {{ .Values.global.postgres.kyoo_migrations.userKey }}
+                  name: {{ .Values.global.postgres.kyoo_migrations.existingSecret }}
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: {{ .Values.global.postgres.kyoo_migrations.passwordKey }}
+                  name: {{ .Values.global.postgres.kyoo_migrations.existingSecret }}
+            - name: POSTGRES_DB
+              value: {{ .Values.global.postgres.database }}
+            - name: POSTGRES_SERVER
+              value: {{ .Values.global.postgres.host }}
+            - name: POSTGRES_PORT
+              value: "{{ .Values.global.postgres.port }}"
+        {{- with .Values.back.extraInitContainers }}
+          {{- tpl (toYaml .) $ | nindent 6 }}
+        {{- end }}
+      containers:
+        - name: main
+          image: {{ .Values.back.kyoo_back.image.repository | default (printf "%s/kyoo_back" .Values.global.image.repositoryBase) }}:{{ default (include "kyoo.defaultTag" .) .Values.back.kyoo_back.image.tag }}
+          imagePullPolicy: {{ default .Values.global.image.imagePullPolicy }}
+          args:
+            {{- with .Values.back.kyoo_back.extraArgs }}
+              {{- toYaml . | nindent 12 }}
+            {{- end }}
+          env:
+            - name: TRANSCODER_URL
+              value: http://{{ include "kyoo.transcoder.fullname" . }}:7666
+            - name: PUBLIC_URL
+              value: {{ .Values.kyoo.address }}
+            - name: UNLOGGED_PERMISSIONS
```

Comment:
You are missing the `REQUIRE_ACCOUNT_VERIFICATION` & the `DEFAULT_PERMISSIONS` env vars.

Reply:
similar to the scanner comment.

I tried not to add every single configuration on the first pass. The power user can set individual environmental variables via back.kyoo_back.extraEnv.

Alternatively we can expose general settings via kyoo.languages or kyoo.unloggedPermissions. Definitely would need some help picking out the most important ones to showcase.

Reply:
Added those settings. closing